sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/NotificationHook/NotificationHook.cs

```diff
@@ -50,7 +50,7 @@ internal static HookInfoPatch GetPatchModel(NotificationHook hook)
             return hook switch
             {
                 EmailNotificationHook h => new EmailHookInfoPatch() { HookName = h.Name, Description = h.Description, ExternalLink = h.ExternalLink, HookParameter = h.HookParameter, Admins = h.Administrators },
-                WebNotificationHook h => new WebhookHookInfoPatch() { HookName = h.Name, Description = h.Description, ExternalLink = h.ExternalLink, HookParameter = h.HookParameter, Admins = h.Administrators },
+                WebNotificationHook h => new WebhookHookInfoPatch() { HookName = h.Name, Description = h.Description, ExternalLink = h.ExternalLink, /*HookParameter = h.HookParameter*/ Admins = h.Administrators },
```

Comment:
Conversion required.

Reply:
you mean with arch board? or v-team?
is there an issue tracking this?

Reply:
I got confused for a little bit. I'm assuming you misread it as "conversation". I mean, we need to convert from one model to another one given that the service team split what we had in two similar models. It's been tracked as part of "UpdateHook" in https://github.com/Azure/azure-sdk-for-net/issues/21177 given that tests won't pass without fixing this.

Reply:
You are right. :)